play/src/components/AnswerSelector.stories.tsx

```diff
@@ -2,7 +2,7 @@ import React from 'react';
 import { ComponentStory, ComponentMeta } from '@storybook/react';
 import { ThemeProvider } from '@mui/material/styles';
 import AnswerSelector from './AnswerSelector';
-import AnswerState from '../lib/PlayModels';
+import { AnswerState } from '../lib/PlayModels';
```

Comment:
PlayModels has more than one model now. Can no longer use default export, typ for all such changes